frontend/src/app/main/ui.cljs

```diff
@@ -29,6 +29,7 @@
 (mf/defc on-main-error
   [{:keys [error] :as props}]
   (mf/with-effect
+    (js/console.log error)
```

Comment:
perhaps console.error?

Reply:
Yep, but this is debug only so no importance here.